core-services/prow/02_config/_plugins.yaml

```diff
@@ -704,32 +704,7 @@ plugins:
   operator-framework/ghostunnel:
   - approve
 
-  containers/libpod:
-  - approve
-  - assign
-  - blunderbuss
-  - blockade
-  - cat
-  - dog
-  - heart
-  - golint
-  - help
-  - hold
-  - label
-  - lgtm
-  - lifecycle
-  - override
-  - retitle
-  - shrug
-  - sigmention
-  - size
-  - skip
-  - trigger
-  - verify-owners
-  - wip
-  - yuks
-
-  containers/buildah:
+  containers:
   - approve
```

Comment:
You might still want individual repos to opt into `approve` if not all of them use OWNERS

Reply:
I think we should force repos to use `OWNERS` if they're using OpenShift Prow - consistency is easier than having per-repo variation.